dp/generate_climos.py

```diff
@@ -583,6 +608,79 @@ def update_metadata_and_time_var(input_file, t_start, t_end, operation, climo_fi
 
     return climo_filepath
 
+def update_generate_climos_history(input_filepath, outdir, operation, convert_longitudes, 
+                                    split_vars, split_intervals, output_resolutions, netCDF_file, time_cdo_format, position=0):
+    """This function takes the start time and end time history-attribute-lines
+    of generate_climos command. It returns a string of the entire history
+    attribute. The result indicates when the generate_climos started and ended.
+
+    :param netCDF_file: (CFDataset) netCDF file to be updated
+    :param time_cdo_format: (str) the point of time that generate_climos history(start or end) was updated
+    :param position: (int) the position that the new history attribute line will be inserted
```

Comment:
That's clever. Maybe explain this slightly more, to the effect that it indexes into the lines of the history attribute.